lib/src/main/scala/spinal/lib/bus/bmb/BmbDecoder.scala

```diff
@@ -90,7 +123,8 @@ case class BmbDecoder(p : BmbParameter,
     }
     for(output <- io.outputs) output.rsp.ready := io.input.rsp.ready
 
-    cmdWait := (rspPending && (hitsS1 =/= rspHits || rspNoHitValid)) || rspPendingCounter === pendingMax
+    cmdWait := ((rspPending && (hitsS1 =/= rspHits || rspNoHitValid))
+               || ((rspPendingCounter ^ pendingMax) & ~U(input.isWrite, log2Up(pendingMax + 1) bits)) === 0)
```

Comment:
I think ((rspPendingCounter ^ pendingMax) & ~U(input.isWrite, log2Up(pendingMax + 1) bits)) is wrong

For instance, let's say :
- isWrite = 1
- rspPendingCounter = 0xFF
- pendingMax = 0x100

the result will say " all good dont wait.
Maybe instead, just do : 
rspPendingCounter === pendingMax || rspPendingCounter === pendingMax-1

That should be good enough.